### ISSUE TITLE
Remove KeePassX and Add KeePassXC

### ISSUE THREAD

**Issue by FK78** [cryptoseb/CryptoPaper#2]
KeePassXC was started as a community fork of KeePassX due to what the community considered to be very slow development of KeePassX, and the unresponsiveness of its maintainer.

**Comment by cryptoseb** [comment]
Yes, thanks for this. Will definitely include it in the commit tonight. Appreciate the support.
